CONTRIBUTING.md

```diff
@@ -4,7 +4,28 @@
 1. All python code must be python 2.6/2.7/3.6 compatible
 1. The code should follow linting from pylint
 
-## Unit tests
+## Developing locally
+
+Dependencies: 
+https://virtualenv.pypa.io/en/latest/installation.html
+
+```bash
+make install  # for centOS 8 dependencies
+CENTOS_VER=7 make install  # for centOS 7 dependencies
+CENTOS_VER=6 make install  # for centOS 6 dependencies
```

Comment:
I don't know about other systems but on Fedora 32 and greater this is not possible.
- python26 is available on Fedora 32 but not on 33 anymore
- even with python26 installed on F32, you can't use it with virtualenv (https://developer.fedoraproject.org/tech/languages/python/multiple-pythons.html)
```
Running virtualenv with interpreter /usr/bin/python2.6
ERROR: this script requires Python 2.7 or greater.
```

Reply:
if you would really wish to have the development env in py26, then you could get py2.6 from the pyenv.
I tested local development in centos6 and CENTOS_6 make install works as expected